front/src/views/exportBankExtractComponent/index.jsx

```diff
@@ -174,27 +172,28 @@ export default class BankExtractModal extends React.Component {
     handleClose() {
       this.setState({file:null})
       this.setState({ showFileModal: false });
-      this.setState({ showExtractModal: true });
+      this.setState({ showExtractModal: false });
+      this.setState({ extractObjects: [] });
     }
   
     handleShow() {
       this.setState({file:null})
       this.setState({ showFileModal: true });
     }
 
-    handleCloseExtract() {
-      this.setState({extractObjects: []})
-      this.setState({ showExtractModal: false });
+    handleList() {
+      this.handleSubmit()
+      this.setState({ showFileModal: false })
+      this.setState({ showExtractModal: true })
```

Comment:
Acredito que dá pra realizar todas essas mudanças em apenas um this.setState
da seguinte forma:
this.setState({x:value, y:value, z:value})